### PR TITLE
nullify tunnel on close sauce connect

### PR DESCRIPTION
**Pull request by wKich**
Fix issue with "Cannot read property 'on' of null" when re-run tests in web ui

---
This change is [Reviewable](https://reviewable.io/reviews/saulis/gemini-sauce/11)

### CODE REVIEW

File: lib/sauce.js [hunk 1]
```diff
@@ -7,9 +7,11 @@ function Sauce() {
   var _tunnel;
 
   function stop(done) {
-    console.log('Closing Sauce Connect.');
+    console.log("Sauce Connect closing tunnel.");
```
Comment (limonte) [inline lib/sauce.js]: Unnecessary line change, please revert.

File: lib/sauce.js [hunk 2]
```diff
@@ -7,9 +7,11 @@ function Sauce() {
...
     if(_tunnel) {
       _tunnel.close(function() {
+        console.log("Sauce Connect closed.");
```
Comment (limonte) [inline lib/sauce.js]: Please follow the project style-guide and use single quotes.

**Review by limonte — CHANGES_REQUESTED** [review]
Thank you @wKich for the PR, a couple of small changes needed:

File: lib/sauce.js (hunk 2 above)
Comment (wKich) [reply lib/sauce.js]: Sorry about that :) Fixed

**Review by limonte — APPROVED** [review]
Nice catch, thanks!

I'll release a new version right away 🚀

**Comment by wKich** [comment]
Thanks a lot. I think you may close #8 because its duplicate

**Comment by limonte** [comment]
https://github.com/Saulis/gemini-sauce/releases/tag/v1.0.2 🎉

**Pull request merged**